Objects/abstract.c

```diff
@@ -1993,6 +1993,9 @@ PySequence_Tuple(PyObject *v)
     if (result == NULL)
         goto Fail;
 
+    // bpo-15108: Code can access the result tuple while being
+    // incomplete when calling PyIter_Next().
+    PyObject_GC_UnTrack(result);
```

Comment:
I would prefer to add a new API to create an untracked tuple: https://bugs.python.org/issue15108#msg387029

Reply:
Let's discuss int the issue, then I can modify the PR :)